src/docs/documentation/references/creating-plugins.mdx

```diff
@@ -70,7 +70,8 @@ A set of preconfigured webpack config plugins
 exports.onCreateWebpackConfig = ({
  stage, getConfig, rules, loaders, actions
 }) => {
-  actions.setWebpackConfig({
```

Comment:
hey, I think you're introducing another typo here, can you get this back?

Reply:
sorry about  that, it's fixed now!